src/components/offers/applyForm/ApplyForm.js

```diff
@@ -48,7 +54,8 @@ export const ApplyForm = () => {
         })
         Swal.showLoading()
         offersAPI.applyWithNoUser(id, name, surname, email, files)
-            .then(() => {
+            .then((result) => {
+                if(!result.ok) throw Error("Something went wrong!")
```

Comment:
Czym się różni throw Error od console.error?

Reply:
console.error wypisuje tylko na konsolę - tak jakbyś zrobił println. Rzucenie errora to tak, jakbyś rzucił exception w Javie

Reply:
Gdzieś go przechwytujemy, czy też w jsie wyjątki nie ubijają aplikacji?